src/content/api/node.mdx

```diff
@@ -42,14 +42,12 @@ import webpack from 'webpack';
 
 The imported `webpack` function is fed a webpack [Configuration Object](/configuration/) and runs the webpack compiler if a callback function is provided:
 
-```js-with-links
+```js
 const webpack = require('webpack');
 
-webpack({
-  // [Configuration Object](/configuration/)
-}, (err, stats) => { // [Stats Object](#stats-object)
+webpack({}, (err, stats) => {
   if (err || stats.hasErrors()) {
-    // [Handle errors here](#error-handling)
+    //
```

Comment:
```suggestion
```

Reply:
It's a placeholder, guess it makes more sense to use `// ...`.